src/test/ui/fn_once-moved.stderr

```diff
@@ -1,12 +1,15 @@
 error[E0382]: use of moved value: `debug_dump_dict`
   --> $DIR/fn_once-moved.rs:21:5
    |
+16 |         for (key, value) in dict {
+   |                             ---- dict moved here
```

Comment:
@tommyip yeah, seems like progress; one thing I would suggest though -- can we merge the two "notes"? e.g., by using "closure cannot be invoked more than once because it moves the variable `dict` out of its environment" as the text for `span_note`?